src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Commands.Resources.Rest.Netcore.csproj

```diff
@@ -17,10 +17,10 @@
   
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
     <DefineConstants>TRACE;DEBUG;NETSTANDARD</DefineConstants>
+    <DelaySign>false</DelaySign>
   </PropertyGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp2.0' ">
-    <PackageReference Include="Hyak.Common" Version="1.1.3" />
```

Comment:
Same comment on depednencies - assuem that this applies across the board for all modules